schema/deploy/todo_create_table.sql

```diff
@@ -0,0 +1,12 @@
+-- Deploy todo_app:todo_create_table to pg
+-- requires: todoAppschema
+
+BEGIN;
+  CREATE TABLE todo_app.todo_create_table (
```

Comment:
a small nitpick on table naming convention:
"todo_create_table" is kind of a confusing name for this table. A table name should describe the data that it contains.
The "thing" that this table contains is a "todo", so I would just name this table "todo"

Reply:
> a small nitpick on table naming convention: "todo_create_table" is kind of a confusing name for this table. A table name should describe the data that it contains. The "thing" that this table contains is a "todo", so I would just name this table "todo"

it makes sense. deal !!

By the way,  I had thought something like: 
todo = what is this table for.
behavior = create , alter or remove.
table = schema table.

Reply:
Your way makes more sense due to sqitch 
has desc on plan through flag -n